asm/overlay_01_021E5900.s

```diff
@@ -8,6 +8,7 @@
 #include "constants/items.h"
 #include "msgdata/msg/msg_0096_D31R0201.h"
 #include "msgdata/msg/msg_0066_D23R0102.h"
+#include "dsprot.h"
```

Comment:
generally not a good idea to include non-constant C headers in asm files
it can break the assembler and makes things messy
I'd rather func definitions be kept in the .inc file for this asm file and the relevant header included when decompiled